geolocator_android/android/src/main/java/com/baseflow/geolocator/permission/PermissionUtils.java

```diff
@@ -38,4 +34,17 @@ public static boolean hasPermissionInManifest(Context context, String permission
 
     return false;
   }
+
+  @SuppressWarnings("deprecation")
+  private static PackageInfo getPackageInfo(Context context) throws PackageManager.NameNotFoundException {
+    final PackageManager packageManager = context.getPackageManager();
+    final String packageName = context.getPackageName();
+
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+      return packageManager.getPackageInfo(packageName,
+          PackageManager.PackageInfoFlags.of(PackageManager.GET_PERMISSIONS));
+    }
+
+    return packageManager.getPackageInfo(packageName, PackageManager.GET_PERMISSIONS);
```

Comment:
Sorry about this, I forgot hitting the safe button with the earlier review. I meant to ask you to update t this statement so that the branched flow contains the deprecated code:

```suggestion
    if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
      return packageManager.getPackageInfo(packageName, PackageManager.GET_PERMISSIONS);
    }
    
    return packageManager.getPackageInfo(packageName,
        PackageManager.PackageInfoFlags.of(PackageManager.GET_PERMISSIONS));
```